ResilientDecoding.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name     = 'ResilientDecoding'
-  s.version  = '1.0.2'
+  s.version  = '1.0.3'
```

Comment:
If we’re adding new functionality (as opposed to fixing a bug), shouldn’t this be 1.1.0?

Reply:
I agree.